doc/parameter_view/parameters.css

```
@@ -36,6 +36,15 @@ ul {
   background-color: #ccc;
 }
 
+
+/* show a + in front of expandable things and a - if it is expanded: */
+.collapsible:not(.active)::before {
+    content: "[+]";
+}
+.active::before {
+    content: "[-]";
```

Can you make this a longer dash so that the plus and minus sines take up the same space?
```suggestion
    content: "[−]";
```

---

Or be explicit: `&ndash;` or `&mdash;`

---

> Or be explicit: `&ndash;` or `&mdash;`

that doesn't work, but @jdannberg had the right idea.